frontend/src/components/AddIngredients.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components/macro";
+import { useState } from "react";
+
+export default function AddIngredients({ onAddClick }) {
+  const [ingredient, setIngredient] = useState("");
+
+  const handeAddClick = () => {
```

Comment:
maybe you want to correct your typo?(hande)

Reply:
Thanks for the review. I will correct the Typo.